src/control/pollingcontrolproxy.h

```diff
@@ -19,7 +19,7 @@ class PollingControlProxy {
     }
 
     PollingControlProxy(const ConfigKey& key, ControlFlags flags = ControlFlag::None) {
-        m_pControl = ControlDoublePrivate::getControl(key, flags);
+        m_pControl = ControlDoublePrivate::getControl(std::move(key), flags);
```

Comment:
getControl does not always keep the key, so moving has no benefit here. Conditionally copy form a const & should work best.